bin/extract_galaxy_tools.py

```diff
@@ -263,7 +264,7 @@ def get_tool_metadata(tool, repo, ts_cat, excluded_tools, keep_tools):
                     metadata['Status'] = 'Up-to-date'
     # get bio.tool information
     if metadata["bio.tool id"] is not None:
-        r = requests.get(f'https://bio.tools/api/tool/{metadata["bio.tool id"]}/?format=json')
+        r = requests.get(f'https://130.226.25.21/api/tool/{metadata["bio.tool id"]}/?format=json')
```

Comment:
Maybe define a variable at the top of the file, e.g. `BIOTOOLS_API_URL = https://130.226.25.21/api/` and reuse it here?

Also, can you add me to the repo please? :)